weblate/trans/tests/test_vcs.py

```diff
@@ -228,6 +228,12 @@ def test_configure_branch(self):
         )
 
 
+class VCSGerritTest(VCSGitTest):
+    _class = GitWithGerritRepository
+    _vcs = 'git'
+    _branch = 'default'
```

Comment:
I think the correct branch for Git is master, default is probably copied from Mercurial testcase...